capstone/src/main/java/com/google/univiz/scorecard/CollegeData.java

```diff
@@ -0,0 +1,31 @@
+package com.google.univiz.scorecard;
+
+/**
+ * CollegeData is an interface that contains methods and functions which communicate with the
+ * CollegeScorecardAPI to obtain college data from the URL in JSON format and parse them to become
+ * ScorecardData java objects.
+ */
+public interface CollegeData {
+
+  /** TODO(biancamacias): decide what getSuggestions does */
+  public List<ScorecardData> getSuggestions(String suggestion) {}
```

Comment:
If this is an interface, since you're not going to provide a default implementation here, then I believe you can do this instead: 
```suggestion
  public List<ScorecardData> getSuggestions(String suggestion);
```
Likewise for the remaining functions.